webpack/assets/javascripts/react_app/components/ModelsTable/ModelsTable.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import URI from 'urijs';
+import { Table } from '../common/table';
+import { STATUS } from '../../constants';
+import MessageBox from '../common/MessageBox';
+import { translate as __ } from '../../common/I18n';
+import schema from './ModelsTableSchema';
+
+class ModelsTable extends React.Component {
+  componentDidMount() {
+    this.props.getTableItems('models', new URI(window.location.href).query(true));
+  }
+
+  render() {
+    const {
+      getTableItems, sortBy, sortOrder, error,
+    } = this.props;
+
+    return this.props.status !== STATUS.ERROR ?
```

Comment:
Yes. I would like to set a delay before the spinner will show. Otherwise I believe the user experience will be bumpy. Because it takes less than 100ms to show the data so the spinner is shown and quickly the table is rendered.

Reply:
thanks to @xprazak2  we could use this once merged  https://github.com/patternfly/patternfly-react/pull/886/